tests/test_utils/test_schemaview.py

```diff
@@ -697,6 +747,145 @@ def test_uris_without_default_prefix() -> None:
     assert view.get_uri("test_slot", imports=True) == "https://example.org/test#test_slot"
 
 
+CREATURE_EXPECTED = {
+    "class": {
+        CREATURE_SCHEMA: {"MythicalCreature", "HasMagic", "MagicalAbility", "Dragon", "Phoenix", "Unicorn"},
+        "creature_basics": {"Entity", "Creature", "Location", "CreatureAttribute", "HasHabitat"},
+    },
+    "slot": {
+        CREATURE_SCHEMA: {"creature_class", "magical_abilities", "level_of_magic"},
+        "creature_basics": {"id", "name", "description", "species", "habitat"},
+    },
+    "type": {
+        CREATURE_SCHEMA: {"MagicLevel"},
+        "creature_types": {"string", "integer", "boolean"},
+    },
+    "enum": {CREATURE_SCHEMA: {"CreatureClass"}},
+    "subset": {CREATURE_SCHEMA: set(), "creature_subsets": {"mythical_creature", "generic_creature"}},
+    "element": {},
+}
+
+# add in the elements
+for value in CREATURE_EXPECTED.values():
+    for s_name, el in value.items():
+        if s_name not in CREATURE_EXPECTED["element"]:
+            CREATURE_EXPECTED["element"][s_name] = set()
+        CREATURE_EXPECTED["element"][s_name].update(el)
+
+
+@pytest.mark.parametrize("schema", ["creature_view", "creature_view_remote", "creature_view_local"])
+@pytest.mark.parametrize("entity", CREATURE_EXPECTED.keys())
+def test_creature_schema_entities_with_without_imports(
```

Comment:
This can replace a number of the existing tests -- I will clean things up in a subsequent PR.